### PR TITLE
tensorflow 1.8.0

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: tensorboard-feedstock/recipe/meta.yaml [hunk 1]
```diff
@@ -7,15 +7,15 @@ package:
...
   url: https://pypi.io/packages/py3/t/tensorboard/tensorboard-{{ version }}-py3-none-any.whl  # [win]
-  sha256: 4c4793f904d2e8f5b83fc05ae3330e48c8305f08a3250019a6f44e37681644a9                    # [win]
+  sha256: 7776cc8bcfd0d07cd106e9e86c011ab8eca38c97b57f5433b9f0fd6bbf31a36e                    # [win]
 
 build:
-  number: 1
+  number: 0
   skip: True  # [not x86_64]
   skip: True  # [win and vc<14]
   # Do not include the tensorboard entry point as it conflicts with a file in
```
Comment (jjhelmus) [inline tensorboard-feedstock/recipe/meta.yaml]: Can we swap this so that tensorboard provides the entrypoint and tensorflow-base does not?

Comment (nehaljwani) [reply tensorboard-feedstock/recipe/meta.yaml]: Won't pip install create the entry point anyway for unix?

Comment (jjhelmus) [reply tensorboard-feedstock/recipe/meta.yaml]: it will but we can remove it, `rm ${PREFIX}/bin/tensorboard`

Comment (nehaljwani) [reply tensorboard-feedstock/recipe/meta.yaml]: Okay, I'll update the PR with those changes, but only for Windows.

Comment (jjhelmus) [reply tensorboard-feedstock/recipe/meta.yaml]: sounds good, I'll update for linux when I do those builds.

Comment (nehaljwani) [reply tensorboard-feedstock/recipe/meta.yaml]: PR updated.

**Comment by jjhelmus** [comment]
Thanks @nehaljwani

**Pull request merged**